src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/RemoveValidator.java

```diff
@@ -16,10 +16,12 @@
 @Component
 public class RemoveValidator {
 
-    public Errors validate(RemoveDto removeDto, Errors errors, String loggingContext) {
-
-        // TODO Check that the filing date is 'null' if an Update block is present. See UAR-1461.
-
+    public Errors validate(OverseasEntitySubmissionDto overseasEntitySubmissionDto, Errors errors, String loggingContext) {
+        var updateDto = overseasEntitySubmissionDto.getUpdate();
```

Comment:
I think the validateFilingDate should happen in validatePartialRemoveDetails, and the RemoveValidator should just validate the fields inside the 'remove' section of the json/dto.

Reply:
validateFull method is already taking the overseasEntitySubmissionDto object though and checking the date, so this approach is already being used.

Very happy to discuss though.

Reply:
Maybe the compromise is to change the implementation and pass in the RemoveDto and the filing date (but not the overseasEntitySubmissionDto)?

Reply:
The more I think about it, the more I think you're right @markpit and we need to change this for both the full validation and partial validation implementation. A call perhaps?

Reply:
yes ok, i didn't realise the full validation was already doing this.

Reply:
There don't seem to be any satisfactory alternatives to this, are you sure we need to do anything here?

Reply:
Had a call and decided to move the filing date check out of the remove validator.